extensions/amp-sticky-ad/amp-sticky-ad.md

```diff
@@ -38,11 +38,12 @@ limitations under the License.
 ## Behavior
 
 - There can be only one `<amp-sticky-ad>` in an AMP document. The `<amp-sticky-ad>` should only have one direct child of `<amp-ad>`.
-- The sticky will appear on the bottom of a page.
+- The sticky ad will appear on the bottom of a page.
 - The height of the sticky-ad is whatever its child needs up to its max-height.
```

Comment:
Also suggest adding : 
"amp-sticky-ad gently introduces a full width blank container and then fills the sticky ad based on the width and height of the amp-ad"